core/src/main/java/de/sesu8642/feudaltactics/lib/ingame/GameController.java

```diff
@@ -30,9 +31,16 @@ public class GameController {
 
 	private final Logger logger = LoggerFactory.getLogger(this.getClass().getName());
 
-	public static final Color[] PLAYER_COLORS = { new Color(0.2F, 0.45F, 0.8F, 1), new Color(0.75F, 0.5F, 0F, 1),
-			new Color(1F, 0.67F, 0.67F, 1), new Color(1F, 1F, 0F, 1), new Color(1F, 1F, 1F, 1),
-			new Color(0F, 1F, 0F, 1) };
+	/** All colors available to bots and user in the game. */
+	public static Color blue = new Color(0.2F, 0.45F, 0.8F, 1);
+	public static Color orange = new Color(0.75F, 0.5F, 0F, 1);
+	public static Color pink = new Color(1F, 0.67F, 0.67F, 1);
+	public static Color yellow = new Color(1F, 1F, 0F, 1);
+	public static Color white = new Color(1F, 1F, 1F, 1);
+	public static Color green = new Color(0F, 1F, 0F, 1);
+
+	public static Color[] colorBank = { orange, blue, pink, yellow, white, green };
```

Comment:
I don't like this global state here.
I think the colors and colorBank should be final. PLAYER_COLORS (better playerColors now) should be non-static.

Reply:
I will correct that by EOD. Thank you.

Reply:
When attempting to make playerColors non static, this is causing issues in the EditorController class because it makes a static reference to the playerColors array via "GameController.playerColors" to be able to generate an empty gamestate. Would you like me to keep playerColors as static or what would you recommend I do for a fix? Thanks.

Reply:
EditorController doesn't really need to know about the color setting. The editor is not near complete anyway. It can use colorBank instead for now.